core/src/main/scala/cats/instances/future.scala

```diff
@@ -1,37 +1,47 @@
 package cats
 package instances
 
-import scala.util.control.NonFatal
 import scala.concurrent.{ExecutionContext, Future}
+import scala.util.control.NonFatal
 
 trait FutureInstances extends FutureInstances1 {
 
   implicit def catsStdInstancesForFuture(
     implicit ec: ExecutionContext
   ): MonadError[Future, Throwable] with CoflatMap[Future] with Monad[Future] =
     new FutureCoflatMap with MonadError[Future, Throwable] with Monad[Future] with StackSafeMonad[Future] {
-      def pure[A](x: A): Future[A] = Future.successful(x)
+      def pure[A](x: A): Future[A] =
+        Future.successful(x)
+
+      def flatMap[A, B](fa: Future[A])(f: A => Future[B]): Future[B] =
+        fa.flatMap(f)
 
-      def flatMap[A, B](fa: Future[A])(f: A => Future[B]): Future[B] = fa.flatMap(f)
+      def handleErrorWith[A](fea: Future[A])(f: Throwable => Future[A]): Future[A] =
+        fea.recoverWith { case t => f(t) }
 
-      def handleErrorWith[A](fea: Future[A])(f: Throwable => Future[A]): Future[A] = fea.recoverWith { case t => f(t) }
+      def raiseError[A](e: Throwable): Future[A] =
+        Future.failed(e)
 
-      def raiseError[A](e: Throwable): Future[A] = Future.failed(e)
-      override def handleError[A](fea: Future[A])(f: Throwable => A): Future[A] = fea.recover { case t => f(t) }
+      override def handleError[A](fea: Future[A])(f: Throwable => A): Future[A] =
```

Comment:
There are no actual changes in this file, right?

Reply:
yes, only formatting